doc/source/conf.py

```diff
@@ -298,8 +303,39 @@
     for page in moved_api_pages
 }
 
+
+common_imports = """\
+.. currentmodule:: pandas
+
+.. ipython:: python
+   :suppress:
+
+   import numpy as np
+   from pandas import *
+   import pandas as pd
+   randn = np.random.randn
+   np.set_printoptions(precision=4, suppress=True)
+   options.display.max_rows = 15
+"""
+
+
+def get_contributors(start, end):
+
+    return announce.build_string('{}..{}'.format(start, end), None)
+
+
+contributors = {
+    'v0.24.0': announce.build_string('v0.23.1..HEAD'),
+    'v0.23.2': announce.build_string('v0.23.1..HEAD'),
+    'v0.23.1': announce.build_string('v0.23.0..v0.23.1'),
+    'v0.23.0': announce.build_string('v0.22.0..v0.23.0'),
+}
```

Comment:
This is run every time we build the docs. It'll probably be run even for single page / docstring ones. I was concerned it would take a while, but it's surprisingly fast.

Once I finish the rest of the whatsnew, I'll time it again and look into ways of speeding it up if is a problem.